tripper/triplestore.py

```diff
@@ -818,20 +836,260 @@ def _add_function_emmo(self, func, expects, returns, base_iri):
         if doc_string:
             self.add((func_iri, DCTERMS.description, en(doc_string)))
 
-        # self.add((func_iri, RDF.type, Task))
-        # self.add((func_iri, RDFS.label, en(name)))
-        # for parname, iri in pars:
-        #    par = f"{func_iri}_input_{parname}"
-        #    self.add((par, RDF.type, DataSet))
-        #    self.add((par, RDFS.label, en(parname)))
-        #    self.add((par, MAP.mapsTo, iri))
-        #    self.add((func_iri, hasInput, par))
-        # for i, iri in enumerate(returns):
-        #    val = f"{func_iri}_output{i+1}"
-        #    self.add((val, RDF.type, DataSet))
-        #    self.add((val, MAP.mapsTo, iri))
-        #    self.add((func_iri, hasOutput, val))
-        # if doc_string:
-        #    self.add((func_iri, DCTERMS.description, en(doc_string)))
-
         return func_iri
+
+    def add_interpolation_source(  # pylint: disable=too-many-arguments
```

Comment:
I don't think we should add this to the production code yet. Let's explore a bit what implications of registering soft sensors will have. We will typically want to add this as part of the domain ontology

Reply:
> I don't think we should add this to the production code yet. Let's explore a bit what implications of registering soft sensors will have. We will typically want to add this as part of the domain ontology

I agree. That is why I made issue #92 and asked to look at it - the idea with that issue is to keep tripper as a common interface to triplestore backends and move the methods working on the triplestore into a separate (sub)package.